build.sbt

```diff
@@ -214,7 +214,7 @@ lazy val commonSettings = Seq(
     Cmd("USER", "1002"),
     Cmd(
       "CMD",
-      s"rm -rf /poduser/work/downloads && java -Xmx2g -Dlog4j.configurationFile=/poduser/log-config/log4j2.properties -jar /opt/${(assembly / assemblyJarName).value}"
+      s"[ -z $$DOWNLOAD_DIR ] || rm -rf $$DOWNLOAD_DIR && java -Xmx2g -Dlog4j.configurationFile=/poduser/log-config/log4j2.properties -jar /opt/${(assembly / assemblyJarName).value}"
```

Comment:
Does this mean that that this new download dir is no longer under `/poduser/work`?

Reply:
It will now be under `/poduser/work/backend/` because that's what we're setting it to in the environment variables on the server. It doesn't really matter where it is though.